packages/better-auth/src/plugins/jwt/adapter.ts

```diff
@@ -2,6 +2,42 @@ import type { BetterAuthOptions } from "@better-auth/core";
 import type { DBAdapter } from "@better-auth/core/db/adapter";
 import type { Jwk } from "./types";
 
+class JwkCreationMutex {
```

Comment:
This does not seem correct. Imaging you have multiple server or serverless environment